ci-operator/config/integr8ly/grafana-operator/OWNERS

```diff
@@ -0,0 +1,16 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- david-martin
+- pb82
+- grdryn
+- HubertStefanski
```

Comment:
I think you have realised now, but these all need to be members of the OpenShift GH org. However, it will get synced from the repo once it's merged, so make sure you have an OWNERS file in there with the correct people listed as approvers.